benchmarks/ekobox/benchmark_evol_pdf.py

```diff
@@ -78,25 +69,28 @@ def benchmark_evolve_single_member(tmp_path, cd, lhapdf_path):
 
 
 @pytest.mark.isolated
-def benchmark_evolve_more_members(tmp_path, cd, lhapdf_path):
-    op = cards.generate_operator(
-        [10, 100], update={"interpolation_xgrid": [1e-7, 0.01, 0.1, 0.2, 0.3]}
-    )
-    theory = cards.generate_theory(0, 1.0)
-    nt, no = compatibility.update(theory, op)
+def benchmark_evolve_more_members(
+    tmp_path, cd, lhapdf_path, theory_card: TheoryCard, operator_card: OperatorCard
+):
+    theory = theory_card
+    theory.order = (1, 0)
+    op = operator_card
+    operator_card.mu0 = 1.0
```

Comment:
```suggestion
    op.mu0 = 1.0
```
I guess? (and similar below)